src/widgets/notes/notes-detailed-summary.component.tsx

```diff
@@ -208,17 +208,21 @@ function NotesDetailedSummary(props: NotesDetailedSummaryProps) {
 
   function displayEmptyNotes() {
     return (
-      <SummaryCard
-        name="Notes"
-        styles={{
-          width: "100%",
-          backgroundColor: "var(--omrs-color-bg-low-contrast)",
-          boxShadow: "none",
-          border: "none"
-        }}
-      >
-        <div className={`${styles.emptyNotes} omrs-bold`}>
-          <p>No Notes are documented</p>
+      <SummaryCard name="Notes" styles={{ width: "100%" }}>
```

Comment:
Might wanna use the [EmptyState](https://github.com/openmrs/openmrs-esm-patient-chart-widgets/blob/master/src/ui-components/empty-state/empty-state.component.tsx) component here.